src/traits.rs

```diff
@@ -1235,6 +1235,21 @@ pub trait SystemExt: Sized + Debug + Default + Send + Sync {
     /// ```
     fn long_os_version(&self) -> Option<String>;
 
+    /// Returns the distribution id as defined by os-release,
+    /// or OS-name if unavailable.
```

Comment:
What do you think about:

```suggestion
    /// Returns the distribution id as defined by os-release,
    /// or [`std::env::consts::OS`].
```

?

Reply:
Done.